### PR TITLE
🚀 feat: add TS to helpers package

### PR DESCRIPTION
**Pull request by matheushdsbr**
[JIRA Issue](https://gympass.atlassian.net/browse/)

## Description 📄

 add TS to helpers package

## Platforms 📲

- [x] Web
- [x] Mobile

## Type of change 🔍

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? 🧪

[Enter the tips to test this PR]

- [ ] Unit Test
- [ ] Snapshot Test

## Checklist: 🔍

- [x] My code follows the contribution guide of this project [Contributing Guide](https://github.com/Gympass/yoga/blob/master/CONTRIBUTING.md)
- [ ] Layout matches design prototype: [FIGMA](https://figma.com/file/YOUR_LINK)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: packages/helpers/src/media.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import { BreakpointType, breakpoints } from '../../tokens/src/global/breakpoints';
+
+import { css } from 'styled-components';
+
+import hide from './hide';
+import { Matcher, Media, Width } from './types';
+
+const availableBreakpoints = Object.keys(breakpoints) as BreakpointType[];
```
Comment (aly-lopes) [inline packages/helpers/src/media.ts:8]: 💅 This feels redundant hence you defined a const `BREAKPOINTS_KEYS` with the same value.
```suggestion
const availableBreakpoints = BREAKPOINTS_KEYS;
```

File: packages/helpers/src/hide.ts [hunk 2]
```diff
@@ -1,11 +1,15 @@
-import tokens from '@gympass/yoga-tokens';
+import { breakpoints, Breakpoints } from '../../tokens/src/global/breakpoints';
```
[the remark this answers is not in the document]
Comment (matheushdsbr) [reply packages/helpers/src/hide.ts:1]: @aly-lopes for your Review! 

No reason, it was a detail I missed, I'll fix it!

File: packages/tokens/src/global/breakpoints.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-type BreakpointType = 'xxs' | 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xxl' | 'xxxl';
+export type BreakpointType = 'xxs' | 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xxl' | 'xxxl';
```
[the remark this answers is not in the document]
Comment (matheushdsbr) [reply packages/tokens/src/global/breakpoints.ts:1]: Good suggestion!

[the remark this answers is not in the document]
Comment (matheushdsbr) [reply packages/tokens/src/global/breakpoints.ts]: Hey, I made a refactor based on your suggestions, what do you think? 👀

**Review by aly-lopes — APPROVED** [review]

**Review by evilamaior — APPROVED** [review]

**Pull request merged**